### DISCUSSION TITLE
Question about pytorch

### DISCUSSION THREAD

**Discussion by LJ-Hao** [openai/whisper#2069] [Q&A]
`parameter logits (Tensor of shape (1, 51864)) of distribution Categorical(logits: torch.Size([1, 51864])) to satisfy the constraint IndependentConstraint(Real(), 1), but found invalid values:
tensor([[nan, nan, nan,  ..., nan, nan, nan]], device='cuda:0')`
I run [whisper_real_time](https://github.com/davabase/whisper_real_time) on a linux platform. But I get this problem, some one helps.

**Comment by LJ-Hao** [comment]
I have tried this [way](https://github.com/openai/whisper/discussions/1068) but it did not work.

**Comment by LJ-Hao** [comment]
`!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm 1.5mm`
I have solved this question, but I get the wrong text.

**Comment by phineas-pta** [comment]
raise issue in the repo u using

**Reply by ulatekh** [reply]
whisper_real_time is just a top-level .py file that drives the usual whisper features.

**Reply by phineas-pta** [reply]
doesnt guarantee error-free code
